pkg/tf2pulumi/convert/tf.go

```diff
@@ -1423,6 +1423,43 @@ func convertBody(sources map[string][]byte, scopes *scopes, fullyQualifiedPath s
 	return newAttributes
 }
 
+// camelCaseObjectAttributes rewrites the attributes of objects to camelCase and returns the modified value.
+// when the input is a list of objects or map of objects, those are modified recursively.
+func camelCaseObjectAttributes(value cty.Value, valueType cty.Type) cty.Value {
+	// handle type object({...})
+	if valueType.IsObjectType() {
+		modifiedAttributes := map[string]cty.Value{}
+		for propertyKey, propertyValue := range value.AsValueMap() {
+			modifiedValue := camelCaseObjectAttributes(propertyValue, propertyValue.Type())
+			modifiedAttributes[camelCaseName(propertyKey)] = modifiedValue
+		}
+		return cty.ObjectVal(modifiedAttributes)
+	}
+
+	// handle type list(object({...}))
+	if valueType.IsListType() && valueType.ElementType().IsObjectType() {
+		modifiedValues := make([]cty.Value, value.LengthInt())
+		for index, element := range value.AsValueSlice() {
+			modifiedValues[index] = camelCaseObjectAttributes(element, valueType.ElementType())
+		}
+
+		return cty.ListVal(modifiedValues)
+	}
+
+	// handle type map(object({...}))
+	if valueType.IsMapType() && valueType.ElementType().IsObjectType() {
+		modifiedAttributes := map[string]cty.Value{}
+		for propertyKey, propertyValue := range value.AsValueMap() {
+			modifiedValue := camelCaseObjectAttributes(propertyValue, propertyValue.Type())
+			modifiedAttributes[propertyKey] = modifiedValue
+		}
+		return cty.MapVal(modifiedAttributes)
+	}
```

Comment:
Don't we need to handle set and tuple types here?

Reply:
Done! sets are handled like lists now ✅